### ISSUE TITLE
Renounce BSC Token Contract Ownership

### ISSUE THREAD

**Issue by cl-tim** [congruentlabs/signata-dao#7]
The BSC SATA token is a BEP20UpgradeableProxy token. That is, it is a token that can be upgraded to add or remove features at any time. Originally the intent was to allow the ability to enable minting supply on BSC if the DAO voted for it. As it has been a year since this contract was created and that feature has not been used, this proposal is to renounce ownership of the BSC token contract so that it can never be upgraded to add any features any more.

The benefit of this outcome is it provides an assurance that the BSC network token can never increase supply, especially should a compromise of keys occur.

The token will still retain the ability to burn supply, if supply reduction on BSC is required in the future.

This will be a snapshot vote as the ETH DAO cannot make modifications on BSC.

**Comment by SATA-OG** [comment]
Sounds like a no-brainer to me, removing a feature we don't need or use to improve security

**Comment by RunningWithTurtles** [comment]
> Sounds like a no-brainer to me, removing a feature we don't need or use to improve security

Second 👍

**Comment by cl-tim** [comment]
Roger that.

Re: the other chains, those token contracts aren't upgradeable contracts. We can renounce ownership of them as well, even though it means nothing.

**Comment by RunningWithTurtles** [comment]
> Roger that.
> 
> Re: the other chains, those token contracts aren't upgradeable contracts. We can renounce ownership of them as well, even though it means nothing.

Thanks for the clarification.  If someone feels strongly about renouncing ownership of the contract on other chains they can submit that proposal.

**Comment by cl-tim** [comment]
Yeah it will only come up if someone requires it, like how Bancor requires a renounced contract.

**Comment by RandySavageCreamOCrop** [comment]
I agree the renouncing is a good idea. I don't see why BSC would require minting while other chains do not. As for renouncing other chains, are there any drawbacks?

**Comment by cl-tim** [comment]
> I agree the renouncing is a good idea. I don't see why BSC would require minting while other chains do not. As for renouncing other chains, are there any drawbacks?

It was purely because adding BSC, at the time, was completely unknown how it would go down.

There aren't any drawbacks to renouncing the other chain contracts, the owner role has zero additional control over the contracts.

**Comment by RandySavageCreamOCrop** [comment]
Alright. We can just leave as is for now unless a need to renounce arises.

**Comment by cl-tim** [comment]
Alright. No counter arguments for this proposal have been raised so we'll open this for voting

**Comment by SataCenturion** [comment]
Probably superfluous at this point but I am in favour of renouncing as required

**Comment by cl-tim** [comment]
Proposal opened. Added renouncing other token contracts too:
https://snapshot.org/#/sata.eth/proposal/0x3c185557c69eb67cfc0b24f580b5859af347a67133aa9b4578cc5c650ffd1715
